components/Common/CrossLink/index.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from './index.module.scss';
+import classNames from 'classnames';
+import LocalizedLink from '@/components/LocalizedLink';
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/solid';
+import { FormattedMessage } from 'react-intl';
+import type { FC } from 'react';
+
+type CrossLinkProps = {
+  type: 'previous' | 'next';
+  text: string;
+  url: string;
+};
+
+const CrossLink: FC<CrossLinkProps> = ({ type, text, url }) => (
+  <LocalizedLink className={styles.crossLink} href={url}>
+    <div
+      className={classNames(styles['header'], {
```

Comment:
nit: we can use the dot notation for the classname property, in the following places too
```suggestion
      className={classNames(styles.header, {
```